terraform/dashboards/Near/chain-signatures/chain_sig_new.json

```diff
@@ -2546,18 +2464,14 @@
           "pluginVersion": "11.4.0-79146",
           "targets": [
             {
-              "datasource": {
```

Comment:
Why are we removing datasource UIDs? I don't know if this will break the dashboards in grafana or not once this is applied.

Reply:
It is specified above, maybe that is enough. Again, it is autogenerated, I think we should test it. Maybe before merging.